.landscaper/gardenlet/example/imports.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: imports.gardenlet.landscaper.gardener.cloud/v1alpha1
+kind: LandscaperGardenletImport
+runtimeCluster:
+  apiVersion: landscaper.gardener.cloud/v1alpha1
+  kind: Target
+  spec:
+    type: landscaper.gardener.cloud/kubernetes-cluster
+    config:
+      kubeconfig: |
+        apiVersion:...
+        # here goes the kubeconfig of the target cluster
+
+gardenCluster:
+  spec:
+    type: landscaper.gardener.cloud/kubernetes-cluster
+    config:
+      kubeconfig: |
+        ---
+        apiVersion: v1
+        kind: Config
+      # ... <please insert your kubeconfig here
+
+# optional image vector. Please check docs/deployment/image_vector.md.
+#imageVectorOverwrite: ""
+
+# optional image vector for the components deployed by the Gardenlet itself. Please check docs/deployment/image_vector.md.
+#componentImageVectorOverwrites: ""
+
+# optional backup provider for the Seed cluster registered by the Gardenlet
+# required when gardenlet.componentConfiguration.seedConfig.backup.secretRef is not set
+# secret is deployed into the virtual Garden cluster
+seedBackup:
+  provider: aws
+  credentials:
+  # Provider-related secret keys
+  #
+  # Please checkout the documentation and example manifests of your provider to find out which data keys
+  # are expected.
+  # E.g AWS:
+#    accessKeyID: base64(access-key-id)
+#    secretAccessKey: base64(secret-access-key)
```

Comment:
1. I would move this below `componentConfiguration`
2. Can we get rid of concrete examples of AWS here?